### DISCUSSION TITLE
Using Firefly III in the UK: Adding Sort Code and Account Number for GBP Transactions

### DISCUSSION THREAD

**Discussion by JamesRMUG** [firefly-iii/firefly-iii#9100] [Firefly III] [answered]
Hi all,

This is probably me being really silly as I have just discovered Firefly, but I am from the UK and we only use IBAN and BIC for international payments. Day to day, we use a Sort Code and Account Number. To categorise my transactions, when adding my different account types, is it best to only populate the account number, or is there a way to add a "Sort Code" field for reference?

Maybe I am missing something here, but from the documentation, I can't seem to find the best way to configure Firefly for use only in the UK, managing one currency: GBP (Great British Pound).

Thank you in advance.

**Comment by JC5** [comment]
Hey, thanks for the suggestion! Sort codes are on my list to add in Firefly III. No ETA yet, I'm afraid.

**Reply by JamesRMUG** [reply]
Hey, thank you @JC5 In the meantime, would you suggest in my circumstances just populating the account numbers and leaving IBAN and BIC blank?

**Reply by JC5** [reply, marked as answer]
Yes, that would work the best :)
